src/main/java/com/google/devtools/build/lib/rules/cpp/CcLinkingHelper.java

```diff
@@ -117,6 +117,7 @@ public CcLinkingOutputs getCcLinkingOutputs() {
   private final CcToolchainProvider ccToolchain;
   private final FdoContext fdoContext;
   private String linkedArtifactNameSuffix = "";
+  private String linkedDLLNameSuffix = "";
```

Comment:
Can we just set the `linkedArtifactNameSuffix` in CcLibrary instead of introducing a new variable?